src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -44,6 +44,7 @@ private void InCanvasSearchControl_Unloaded(object sender, RoutedEventArgs e)
             {
                 Application.Current.Deactivated -= currentApplicationDeactivated;
             }
+            MembersListBox.ItemsSource = null;
```

Comment:
yes, this can help remove a binding to a non Observable collection - for example this MembersListBox element is bound to the Ienumerable `FilteredResults` - bindings like this can leak.

Reply:
please see the collection leak section:
https://blog.jetbrains.com/dotnet/2014/09/04/fighting-common-wpf-memory-leaks-with-dotmemory/

Reply:
yes, we can, I'll file a task.